src/Bicep.Core/Tracing/DiagnosticOptionsExtensions.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Azure.Core;
+using System.Collections.Immutable;
+
+namespace Bicep.Core.Tracing
+{
+    public static class DiagnosticOptionsExtensions
+    {
+        private static readonly ImmutableArray<string> ArmClientAdditionalLoggedHeaders = new[]
```

Comment:
>ArmClientAdditionalLoggedHeaders [](http://example.com/codeflow?start=55&length=32)

The SDK doesn't support enabling or disabling logging. Adding headers to these lists prevents the SDK from logging them as `REDACTED`. The SDK includes a default list of headers that it logs, so we don't need this list to be complete.